src/test/java/com/SAS/LeagueManagement/assignRefereesToLeagueInSpecificSeasonBottomUp.java

```diff
@@ -0,0 +1,38 @@
+package com.SAS.LeagueManagement;
+
+import com.SAS.League.League;
+import com.SAS.League.Season;
+import com.SAS.User.Referee;
+import com.SAS.User.Registered;
+import com.SAS.crudoperations.CRUD;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+
+public class assignRefereesToLeagueInSpecificSeasonBottomUp {
+    public static boolean isLeagueExistStub(League league) {
+        return CRUD.isLeagueExist(league);
+    }
+    public static boolean isSeasonExistStub(Season season) {
+        return CRUD.isSeasonExist(season);
+    }
+    public static boolean addRefereesToLeagueInSeasonStub(League league, Season season, List<Referee> referees) {
+    return CRUD.addRefereesToLeagueInSeason(league,season,referees);
+    }
+
+    @Test
+    public void initLeagueTester() {
+        League league = new League("Ligat Ha'al");
+        Assertions.assertTrue(isLeagueExistStub(league));
```

Comment:
Whats the difference between this and assertTrue(.....)?
We used assertTrue and imported import static org.junit.Assert.*;